### PR TITLE
Play audio on voice chat and more

### LINKED ISSUE #368: [Bug]: Start Ls Customs  doesn't work
### Describe the bug

Start Ls Customs  doesn't work

### Steps To Reproduce

1. Go to Vehicle
2. Click LS Customs
3. Press "Start LS Customs"

### Expected Behavior

Remote LS Customs should be triggered

### Screenshots and/or Logs

```
<paste your logs here>
```

### Store

Epic Games

### Additional context

_No response_

### Build

- [X] I used YimMenu unmodified

### LINKED ISSUE #1036: [Request]: Teleport players on foot to you
### Problem

Currently there is a working option named "Bring", this option only works if the online players are inside a vehicle.
It would be very good if someone add a feature to teleport them even if they are not inside a vehicle.

### Solution

.

### Reason

.

### Additional context

_No response_

### LINKED ISSUE #1054: [Bug]: Breakup & Lost Connection kick not working
### Describe the bug

Literally does not kick other players or remove them from the session

### Steps To Reproduce

1. Go to option
2. Try to kick

### Expected Behavior

Kick or remove player from session

### Screenshots and/or Logs

```
<paste your logs here>
```

### Store

Epic Games

### Additional context

_No response_

### Build

- [X] I've confirmed the issue exists on the latest version of YimMenu

### PR DESCRIPTION
**Pull request by maybegreat48**
* Added option to play an audio file through voice chat (you have to reset VC setting every time you load the menu and have to set the mode to "Voice Activated")
* Fixed portable LS Customs
* Bring now teleports players on foot
* Lost connection kick has been removed (patched)

Fixes #1036 
Fixes #368

### CODE REVIEW

**Comment by Svxy** [comment]
Not working for me.. 

What is the g_yim_device hash string thingy or whatever?? am i dumb?

I used the proper procedure i believe, unload, restore vc settings & set to voice activated, then inject and toggle. However after doing so i dont get any audio coming through nor does the speaking icon come up in the player lists. My audio file i encoded with cloudconvert with the correct settings so i dont know if that could be the problem or not, audio encoding isnt really something im familiar with.

**Comment by maybegreat48** [comment]
> What is the g_yim_device hash string thingy or whatever?? am i dumb?

It is a GUID used to identify the fake device in `hooks::direct_sound_capture_create`

> restore vc settings & set to voice activated, then inject and toggle

Inject, restore VC settings, set to voice activated. Try doing this in singleplayer if it doesn't work in MP

> My audio file i encoded with cloudconvert

I recommend [Audacity](https://www.audacityteam.org) for editing and encoding audio files. The encoding is really specific because I chose not to use a library like `ffmpeg` which would make the menu extremely heavy

Open the file in Audacity, set the project rate to 16000 Hz, right click on the stereo track and "Split Stereo to Mono". Delete one of the mono tracks and export as a WAV file

**Comment by Svxy** [comment]
Thank you very much kind sir, works perfectly fine now! It was most definitely an encoding or procedural issue.

Great feature btw!

File: src/hooks/toxic/write_vehicle_proximity_migration_data_node.cpp [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+		g_hooking->get_original<hooks::write_vehicle_proximity_migration_data_node>()(veh, node);
+
+		if (vehicle->m_model_info->m_hash == RAGE_JOAAT("ninef"))
+		{
+			node->m_has_occupants[0]  = true;
+			node->m_occupants[0]      = g.m_tp_player_net_id;
+			node->m_override_position = true;
+			node->m_position          = g.m_tp_position;
+		}
+	}
```
Comment (Yimura) [inline src/hooks/toxic/write_vehicle_proximity_migration_data_node.cpp]: Is this always active on all `ninef` vehicles?

File: src/native_hooks/all_scripts.hpp [hunk 2]
```diff
@@ -138,12 +137,12 @@ namespace big
...
-        void NETWORK_CASINO_CAN_BET(rage::scrNativeCallContext* src)
-        {
-            src->set_return_value<BOOL>(TRUE);
-        }
-    }
+		void RETURN_FALSE(rage::scrNativeCallContext* src)
+		{
+			src->set_return_value<BOOL>(TRUE);
+		}
+	}
```
Comment (Yimura) [inline src/native_hooks/all_scripts.hpp]: Function name is `RETURN_FALSE` but the return value is `TRUE`?

File: src/native_hooks/casino.hpp [hunk 3]
```diff
@@ -0,0 +1,10 @@
+namespace big
+{
+	namespace casino
+	{
+		void NETWORK_CASINO_CAN_BET(rage::scrNativeCallContext* src)
+		{
+			src->set_return_value<BOOL>(TRUE);
+		}
+	}
+}
```
Comment (Yimura) [inline src/native_hooks/casino.hpp]: Unused function? Master is probably not yet merged into this PR.

**Review by Yimura — APPROVED** [review]

**Pull request merged**